Patches/WWII Soviet Faction/WWIISovietFaction_CE_Patch_Clothing.xml

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationSequence">
+		<success>Always</success>
+		<operations>
+
+			<li Class="CombatExtended.PatchOperationFindMod">
+				<modName>WWII Soviet Faction (1.0)</modName>
+			</li>
+
+			<!-- ========== M73 Uniform ========== -->
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="Apparel_M73_Uniform"]/statBases</xpath>
+				<value>
+					<Bulk>8</Bulk>
+					<WornBulk>4</WornBulk>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Apparel_M73_Uniform"]/statBases/ArmorRating_Sharp</xpath>
+				<value>
+					<ArmorRating_Sharp>0.3</ArmorRating_Sharp>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Apparel_M73_Uniform"]/statBases/ArmorRating_Blunt</xpath>
+				<value>
+					<ArmorRating_Blunt>0.075</ArmorRating_Blunt>
+				</value>
+			</li>	
+
+			<!-- ========== Harness ========== -->
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="Apparel_M73_Harness"]/statBases</xpath>
+				<value>
+					<Bulk>3</Bulk>
+					<WornBulk>3</WornBulk>
+				</value>
+			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="Apparel_M73_Harness"]/equippedStatOffsets</xpath>
+				<value>
+					<CarryBulk>18</CarryBulk>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Apparel_M73_Harness"]/apparel</xpath>
+				<value>
+					<apparel>
+						<bodyPartGroups>
+							<li>Shoulders</li>
+						</bodyPartGroups>
+						<layers>
+							<li>Webbing</li>
+						</layers>
+						<tags>
+							<li>WWII_Soviet_Harness</li>
+						</tags>
+						<wornGraphicPath>Things/Apparel/Belts/M73Harness</wornGraphicPath>
+					</apparel>
+				</value>
+			</li>
+
+			<!-- ========== M35 Uniform ========== -->
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="Apparel_M35_Uniform"]/statBases</xpath>
+				<value>
+					<Bulk>8</Bulk>
+					<WornBulk>4</WornBulk>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Apparel_M35_Uniform"]/statBases/ArmorRating_Sharp</xpath>
+				<value>
+					<ArmorRating_Sharp>0.3</ArmorRating_Sharp>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Apparel_M35_Uniform"]/statBases/ArmorRating_Blunt</xpath>
+				<value>
+					<ArmorRating_Blunt>0.075</ArmorRating_Blunt>
+				</value>
+			</li>
+
+			<!-- ========== Vest ========== -->			
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="Apparel_SN-42Armor"]/statBases</xpath>
+				<value>
+					<StuffEffectMultiplierArmor>2</StuffEffectMultiplierArmor>
```

Comment:
Because the mod's recipe for SN-42 Armor specifies Steel and Cloth, it's not considered to be made of "stuff," so there's nothing to multiply this against. Unless I'm mistaken, this can be removed.

Reply:
I'll nuke this line.

Reply:
Fixed in 622b2e0